data/human/transport missions.txt

```diff
@@ -3173,17 +3173,17 @@ mission "Earth Retirement"
 	
 	on offer
 		conversation
-			`As you wander through the spaceport, you see an elderly man and woman sitting on a bench. They both seem to be talking about something that is causing them some amount of stress.`
+			`As you wander through the spaceport, you see an elderly man and woman sitting on a bench, talking about something that seems to be causing them some amount of stress.`
 			choice
-				`	"Who are you two?"`
+				`	(Ask them what the problem is.)`
 				`	(Ignore them.)`
 					decline
-			`	"My name is Donna, and his name is Charles," she says. "We were yard workers here on <origin> for nearly fifty years, but we're both starting to get too old for our jobs. So, we've made the hard decision to retire. We've already chosen a location on the best place we could afford, which... well, happened to be on Earth, so Charles and I are looking for someone to transport us."`
-			`	As Donna talks, Charles begins to frown. "I suppose you could put it like that."`
-			`	"Is that wrong?" Donna replies, who appears to be bracing for the response.`
+			`	You inquire as to what's troubling them, and the woman begins telling you her life's story. "We were yard workers here on <origin> for nearly fifty years, but we're both starting to get too old for our jobs. So, we've made the hard decision to retire. We've already chosen a location on the best place we could afford, which... well, happened to be on Earth, so Charles and I are looking for someone to transport us. Oh, where are my manners? I'm Donna. Pleased to meet you."`
+			`	Donna was clearly looking for someone besides her husband to talk to. But Charles begins to frown. "I suppose you could put it like that."`
```

Comment:
```suggestion
			`	Donna was clearly looking for someone besides her husband to talk to, but Charles begins to frown. "I suppose you could put it like that."`
```
The pause caused by the period feels a little too long when I read it to myself, but up to you.

Reply:
Yeah, upon re-reading, I think you're right.